app/models/concerns/profile_tailoring.rb

```diff
@@ -23,4 +33,27 @@ def added_rules
   def removed_rules
     parent_profile.rules - rules
   end
+
+  def update_os_minor_version(version)
+    return unless version && os_minor_version.empty?
+
+    update!(os_minor_version: version)
+  end
```

Comment:
nitpick (read: optional, code style related): it was less lines the way it was written before.

IMO this can stay here in profile_tailoring, since setting os_minor_version is only ever used for tailoring :+1: